tensorboard/webapp/metrics/effects/metrics_effects_test.ts

```diff
@@ -89,8 +89,52 @@ describe('metrics effects', () => {
       selectors.getMetricsTooltipSort,
       TooltipSort.ALPHABETICAL
     );
+
+    overrideTagMetadata();
+    overrideRunToEid();
   });
 
+  function overrideTagMetadata() {
+    store.overrideSelector(selectors.getMetricsTagMetadata, {
+      scalars: {
+        tagDescriptions: {} as any,
+        tagToRuns: {
+          tagA: ['run1'],
+          tagB: ['run2', 'run3'],
```

Comment:
There are no references to 'run2' thru 'run6' or 'tagC' and 'tagD' or 'defaultExperimentId' anywhere else in this test as far as I can tell.

Maybe just mock the minimum amount of data you need for existing tests to pass and override at a more detailed level only for your new tests?

Reply:
I added some additional data to the state to ensure it did not result in additional requests.

Reply:
Could you add some comments acknowledging which data is unnecessary and why you include it?

Reply:
After your last comment I added some additional tests and ensured that all of the data is being used. In particular the test `does not send requests to experiments lacking a cards tag` references every tag.